Content.Server/GameObjects/Components/HandheldRadioComponent.cs

```diff
@@ -79,12 +78,38 @@ public bool UseEntity(UseEntityEventArgs eventArgs)
 
         public void HeardSpeech(string speech, IEntity source)
         {
-            PassOnMessage(speech);
+            if (RadioOn)
+            {
+                Broadcast(speech, source);
+            }
         }
 
         public int GetListenRange()
         {
             return _listenRange;
         }
+
+        public void Receiver(string message, int channel, IEntity speaker)
+        {
+            if(RadioOn)
+            {
+                Speaker(message);
+            }
+        }
+
+        public void Broadcast(string message, IEntity speaker)
+        {
+            _radioSystem.SpreadMessage(this, speaker, message, _broadcastChannel);
+        }
+
+        public List<int> GetChannels()
+        {
+            return _channels;
+        }
```

Comment:
This should be an auto-property instead ({get; set;})
I also mentioned it below but this should be public IReadOnlyCollection<int> Channels => _channels;